examples/petstore/api/__init__.py

```diff
@@ -0,0 +1,30 @@
+import typing as t
+
+from axion import response
+
+
+async def list_pets(
+        tags: t.Optional[list] = None,
```

Comment:
it actually should be `t.Optional[t.List[str]]` but "axion" decided that it should be `t.Optional[list]` which seems dumb.
Will create issue for that.